blog/2023-09-20-magazine-005.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Inflight Magazine no. 5
+description: Updates from Winglang
+authors: 
+  - dcahana2
+tags: [cloud-oriented programming, winglang, changelog, newsletter,]
+hide_table_of_contents: true
+---
+> The 5th issue of the Wing Inflight Magazine.
+> <!--truncate-->
+
+Hey folks!
+
+We’re back with another overview of our CHANGELOG - a new segment in our quirky Twitch show called [The Wingly Update](https://www.youtube.com/playlist?list=PL-P8v-FRassZBWsNoSafL_ReO0JO0xJVm) where we share news about new features to Wing, so we decided to share the same updates here as well. Let us know what you think!
+
+- 🚀 [New: Variadic Arguments](#variadic-arguments)
+- 🚀 [New: Added set API to MutArray](#added-set-api-to-mutarray)
+- 🔥 [Runtime checked Json => Struct conversion](#runtime-checked-json)
+- 🔥 [Json implicit struct casting](#json-implicit-struct-casting)
+- 🚀 [New: Added addFile API to Bucket](#added-addfile-api-to-bucket)
+- 🚀 [New: Wing Libraries](#wing-libraries)
+- 🔥 [Updated Directory Modules](#directory-modules)
+- 🚀 [New: fs Namespace](#fs-namespace)
+- 🔥 [Breaking Changes](#additional-breaking-changes)
+- 🔥 [New Features](#additional-new-features)
+- 📅 [Upcoming Events](#upcoming-events)
```

Comment:
Can we add a section with links to all of the Wingly shorts we created so far?

Reply:
> Can we add a section with links to all of the Wingly shorts we created so far?

done @eladb